mypy/stubgen.py

```diff
@@ -1305,9 +1305,7 @@ def is_private_member(self, fullname: str) -> bool:
         parts = fullname.split(".")
         return any(self.is_private_name(part) for part in parts)
 
-    def get_str_type_of_node(
-        self, rvalue: Expression, can_infer_optional: bool = False, can_be_any: bool = True
-    ) -> str:
+    def get_str_type_of_node(self, rvalue: Expression, can_be_incomplete: bool = True) -> str:
```

Comment:
While I was here, I renamed `can_by_any` to `can_be_incomplete`.

Reply:
Could you also make can_be_incomplete keyword-only?